package.json

```diff
@@ -15,7 +15,7 @@
     "release-it": "echo \"Running release-it via yarn breaks publishing! Use npx or a Volta global installation.\""
   },
   "volta": {
-    "node": "14.19.1",
+    "node": "16.17.1",
```

Comment:
Ignore this; I'll pull it out of this and check it in on a separate PR and just make sure CI still tests—I spent another 15 minutes yesterday trying to figure out whyyyyyy I couldn't get anything to work with Vitest before remembering that it was the "bump to Node 16 so ESBuild works natively" dance.